cmd/openshift-tests/e2e.go

```diff
@@ -120,8 +120,6 @@ var staticSuites = testSuites{
 				}
 				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
 			},
-			// etcd's vertical scaling test is expensive
-			TestTimeout:         60 * time.Minute,
```

Comment:
Ah, it's the removal of the 60 minute timeout that's causing this test to fail.  `services when running openshift ipv4 cluster ensures external ip policy is configured correctly on the cluster` averages 12 minutes to complete, with the longest successful run taking 25 minutes.

Maybe we need to set TestTimeout to 30 or 35?

Reply:
Well spotted. I've bumped the timeout for serial to 40 mins.